css/index.css

```diff
@@ -143,4 +206,141 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+
+
+
+.aboutText{
+    display: flex;
+    padding:15px;
+    border-bottom: 1px dashed black;
+    margin: 0 0 20px 0;
+    font-size: 15px;
+    line-height: 22px;
+}
+
+
+
+.strat{
+    width: 50%;
+    margin: 0 10px 50px 0px;
+    border-bottom: 1px dashed black;
+    padding: 10px 15px 20px 0px;
+    margin-right: 15px;
+}
+
+
+.work{
+    width: 50%;
+    margin: 0 10px 50px 10px;
+    border-bottom: 1px dashed black;
+    padding: 10px 1px 0px 0px;
+}
+
+
+.middleAbout{
+    display: flex;
+    justify-content: center;
+    margin: 0px 60px 10px 10px;
+    /* flex-wrap: wrap; */
+    align-items: baseline;
+}
+
+.places{
+    /* display: flex; */
+    width: 50%;
+    /* align-items: column; */
+    padding: 15px 15px 20px 0px;
+    border-bottom: 1px dashed black;
+    margin: 0 20px 0px 90px;
+    margin-left: 70px;
+    
+}
+
+
+
+.places p{
+    width: 99%;
+}
+
+.collab{
+    width: 50%;
+    margin: 0 60px 60px 0px;
+    border-bottom: 1px dashed black;
+    padding: 10px 1px 20px 0px;
+    margin-left: 20px;
+    
+}
+
+
+button {
+    display: block;
+	background-color: white;
+	color: black;
+	border-radius: 5px;
+	margin-top: 10px;
+	width: 40%;
+	padding: 7px 5px;
+	font-size: 17px;
+	text-align: center;
+    font-style: bold;
+    /* border-color: black; */
+    border: 3px solid black;
+}
+
+.midS{
+    display: flex;
+    flex-direction: wrap;
+    /* justify-content: space-around; */
+    justify-content: center;
+     
+}
+
+.midS h1{
+    padding: 10px 0 0 0;
+}
+
+.collab h1{
```

Comment:
You have the same padding on a lot of h1s. You could create a class for that to put on h1s that need it or you could give that padding to all h1s rather than listing each separately like this